### ISSUE TITLE
Reactions are not vocalized.

### ISSUE THREAD

**Issue by cyberbaloo** [numerique-gouv/meet#768]
## Bug Report

**Problematic behavior**
Reactions can be accessed using the keyboard but are not vocalized by assistive technologies. 

**Expected behavior/code**
`<button data-attr="send-reaction-clapping-hands" type="button" tabindex="0" data-react-aria-pressable="true" aria-label="Envoyer la réaction clapping-hands" class="d_flex jc_center ai_center trs_background_200ms,_outline_200ms,_border-color_200ms cursor_pointer bd_1px_solid_transparent [&amp;[data-disabled]]:cursor_default gap_0.5rem bdr_4 px_var(--square-padding) py_var(--square-padding) --square-padding_{spacing.0.25} bg-c_transparent fw_medium! c_white [&amp;[data-hovered]]:bg-c_primaryDark.100 [&amp;[data-pressed]]:bg-c_primaryDark.700 [&amp;[data-pressed]]:c_primaryDark.100 [&amp;[data-selected]]:bg-c_primaryDark.700 [&amp;[data-selected]]:c_primaryDark.100 " data-rac="" id="react-aria87198875-:r6t:"><img src="/assets/reactions/clapping-hands.png" alt="" class="min-h_28px min-w_28px pointer-events_none us_none"></button>`

**Steps to Reproduce**
1.Join the meeting,
2. click on the "Envoyer une réaction" button
3. Select an emoji. 

**Possible Solution**
Replace images with emojis and see how this can be vocalized.
If the button is activated several times, vocalize it only once so as not to overload the assistive technology. 

**Additional context/Screenshots**

<img width="605" height="109" alt="Image" src="https://github.com/user-attachments/assets/03f65b9c-8307-4b53-a5de-55a2f62592fa" />

**Comment by lebaudantoine** [comment]
Hi @cyberbaloo,
Thank you for raising this issue. Based on how other proprietary solutions handle similar scenarios, I believe that having reactions vocalized by default may not be ideal. Instead, this behavior should be configurable.

Users should have the ability to customize how assistive technologies interact with and announce reactions.
I'll give a look to the PR #770, and try to ship it as soon as possible.

**Comment by cyberbaloo** [comment]
Good !
